run_dir/static/js/project_samples.js

```diff
@@ -1013,9 +1013,10 @@ function load_charon_summary(){
       if (data['tot'] != 0){
           $('#charon-status').show();
           $('#charon-status-tot').text(data['tot']);
+          $('#charon-status-ab').text(data['ab']);
```

Comment:
Does `aborted` really have to be shortened to `ab`? Doesn't seem to save much typing and is less obvious. Same feeling about all of the others too to be honest. What is `hge`? What is `runn`?